porting/build_and_boot/H9_gitlab_build.rst

```diff
@@ -1,10 +1,12 @@
 
 .. _Gitlab-CI:
 
-Halium-9.0 - Using Gitlab CI scripts
-====================================
+Halium-9.0 - Standalone kernel method
```

Comment:
OK. Here I want to keep the title because:
When discussing with Nikita, after starting to write about standalone kernel builds, he made a clear distinction between this and the Gitlab CI scripts build, as he called this. The difference is that with the former, you build a linux kernel (not halium-boot.img). You then need to manually follow a few steps to construct the halium-boot.img from the kernel. Then you use this together with the GSI in the same manner as you would if you had built the halium-boot.img in the standard way in the first place. After trying out the other method, which he recommended, I discovered that the chances of straying off the path and making mistakes was far smaller. So with this method, although you are just downloading the kernel source, you are actually building more than the standalone kernel.

Reply:
mhm. well. I wonder whether that distinction is relevant enough. Since we're only describing one method for only kernel anyway, I would suggest to focus on the main aspect: just a kernel, rather than on particular technical aspect (it uses gitlab). I mean there are always more variants.(If I remember right Alfred used AOSP instead of Lineage). But ok. My main point is use a consistent name all through out. It took me a while to figure out that the third method mentioned at the start actually WAS the CI method. When I first read it I thought that the CI was an extra fourth one, but then I was lost because I couldn't find the third one anymore. Can you suggest a single name to be used everywhere?

Reply:
After re-reading this and thinking it through, I agree with your suggestions. The Gitlab-CI script vs. standalone kernel distinction is something the devs see, but which the average porter does not need to be concerned with, as long as we don't describe both anyway. 

However, I also noticed that in my PR this summer, I forgot to edit the introduction where the methods are briefly described. This is irrelevant now, but that part needs to be adjusted a little to bring it in line with the rest, if we are going to go with 'full system image build', 'halium-boot method' and 'standalone kernel method'.

The important thing to communicate, as I see it, is that Halium-9.0 can use all three methods, whereas Halium-7.1 can only use the first. And if the full system image build method is only described for halium-7.1 (detailing how to download this version of the source), can this lead to confusion if a halium-9.0 porter chooses to build the full system image?